lib/reporting/mfa_report.rb

```diff
@@ -118,7 +118,7 @@ def query
     def cloudwatch_client
       @cloudwatch_client ||= Reporting::CloudwatchClient.new(
         num_threads: @threads,
-        ensure_complete_logs: true,
+        ensure_complete_logs: false,
```

Comment:
the query in this file uses `| stats` so I think the "split over time and combine" approach is not applicable here